mlflow/utils/os.py

```diff
@@ -0,0 +1,8 @@
+import platform
+
+
+def is_windows():
+    """
+    :return: Returns true if the local system/OS name is Windows.
+    """
+    return platform.system().lower() == "windows"
```

Comment:
I don't think this is a reliable check for all Windows systems.

According to https://stackoverflow.com/questions/1325581/how-do-i-check-if-im-running-on-windows-in-python, some Windows systems might use `win32` or `win64`.

Can we use `os.name == 'nt'` instead?